CIP-xxxxx/README.md

```diff
@@ -0,0 +1,218 @@
+---
+CIP: ???
+Title: Web-Wallet Bridge - Plutus based wallets
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors: NA
+Discussions:
+Created: 2023-10-12
+License: CC-BY-4.0
+---
+
+
+## Abstract
+
+This document describes a CIP-30 extension allowing webpages (i.e. dApps) to interface with Cardano Plutus-wallets. This document is a work in progress and is not yet finalized. It is expected to be updated as the ecosystem evolves. 
+
+## Motivation: why is this CIP necessary?
+
+In order to facilitate future dApp development, we will need a way for dApps to communicate with plutus wallets, given the unique complexities of plutus based addresses. Special provisions need to be made to make the connector compatible with them.  
+
+Specifically, apps building transactions need to be able to get the following information from the wallet:
+
+- Script descriptor
+- ScriptRequirements
+- Change Datum
+
+Additionally, apps need to be able to submit a transaction to the wallet for signing in an asynchronous manner, as gathering of signatures can take a long time and each wallet provider will have its own way of handling this process. 
+
+Finally, the signData() endpoint will have to be disabled when using this extension since they are not compatible with plutus based addresses. plutus contracts cannot sign data, only validate transactions.
```

Comment:
```suggestion
Finally, the `signData()` endpoint will have to be disabled when using this extension since they are not compatible with plutus based addresses. plutus contracts cannot sign data, only validate transactions.
```
... and throughout, whenever functions or parameters are quoted?

Reply:
resolving: updated as suggested.